pkg/renderer/kubernetes.go

```diff
@@ -38,6 +39,8 @@ const (
 	scannerTLSOnly
 	// centralDBOnly renders only the central db
 	centralDBOnly
+	// scannerV4TLSOnly renders only the scanner v4 tls secrets.
+	scannerV4TLSOnly
```

Comment:
unrelated, but I guess we won't have a mode that allows Scanner v4 only?

Reply:
Don't believe thats needed unless we want to create a separate `roxctl scanner generate` command specific for scanner v4.  https://github.com/stackrox/stackrox/pull/9619 adjusted the output of `roxctl scanner generate` to produce both the scanner v2 and v4 files.